nais/dev-fss.json

```diff
@@ -22,7 +22,8 @@
     "SLETTE_DOKUMENT_SCOPES": "97f0b1bc-6aa9-4d44-a3c7-60b4318fbec4/.default",
     "JOURNALFORE_SCOPES": "b32ae17c-0276-4006-9507-4ef49e0e5e20/.default",
     "KAFKA_BOOTSTRAP_SERVERS": "b27apvl00045.preprod.local:8443,b27apvl00046.preprod.local:8443,b27apvl00047.preprod.local:8443",
-    "KAFKA_AUTO_OFFSET_RESET": "latest"
+    "KAFKA_AUTO_OFFSET_RESET": "latest",
```

Comment:
hvorfor brukes latest i dev og none i prod i utgangspunket?

Reply:
Gjetter at det er fordi det ikke er krise om vi mister noen meldinger i dev. @ramrock93 hadde kanskje en tanke bak det når vi sett det?

Reply:
Det ble satt til latest fordi det ikke eksisterte en offset for topicen da den ble deployet. Den fortsatte å klage til det ble satt latest. Nå er det ingen problem om det settes til none.